CONTRIBUTING.md

```diff
@@ -134,13 +135,20 @@ Most contributors will likely perform the following steps when implementing new
 - `./components/{component-name}.json` - Determine if the component variables have been implemented at the top level by finding a file that contains the **components name**.
   - If so, add the neceesary token data here as references to `./theme-data/{dark | light}/{dark-common | light-common}.json`.
   - If not, proceed to the next item in this list.
-- `./theme-data/{dark | light}/{dark-common | light-common}.json` - Determine if the common component variables include the data needed.
+- `./theme-data/{dark | light}/{dark-common | light-common}.json` - Determine if the common theme variables include the data needed.
```

Comment:
I am not completely understanding the process with these reference-tokens just from these docs. Could all of this be automated? As a dev of momentum I would not like to do that manually and as a contributor of the token repo manual checks and tasks are also not efficient. thanks

Reply:
resolved in call